ormar/queryset/queryset.py

```diff
@@ -1049,25 +1053,25 @@ async def bulk_create(self, objects: List["T"]) -> None:
         :param objects: list of ormar models already initialized and ready to save.
         :type objects: List[Model]
         """
-        ready_objects = []
-        for objt in objects:
-            new_kwargs = objt.dict()
-            new_kwargs = objt.prepare_model_to_save(new_kwargs)
-            ready_objects.append(new_kwargs)
+        ready_objects = [
+            obj.prepare_model_to_save(obj.dict())
+            for obj in objects
+        ]
+        expr = self.table.insert().values(ready_objects)
 
-        expr = self.table.insert()
-        await self.database.execute_many(expr, ready_objects)
+        # shouldn't use the execute_many, it's `queries.foreach(execute)`
+        await self.database.execute(expr)
 
-        for objt in objects:
-            objt.set_save_status(True)
+        for obj in objects:
+            obj.set_save_status(True)
```

Comment:
When now I think about this, this is not true, as objects don't have database side populated columns refreshed (so you cannot use this object as a factual representation of a db state). At the same time it actually was saved, but not fetched, need to think about this.